deployment/provider-azure-installer.yaml

```diff
@@ -47,6 +47,11 @@ spec:
             limits:
               cpu: 50m
               memory: 100Mi
+          securityContext:
```

Comment:
Should we be also setting `allowPrivilegeEscalation: false`?

Also `readOnlyRootFilesystem: true`. But not sure if this impedes provider's ability to write secrets on FS

Reply:
I think we should open an issue for this and track it for the next release to ensure we get enough test results with the changes. WDYT?

Reply:
Sounds good.

Reply:
Created https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/537